apps/src/code-studio/pd/professional_learning_landing/landingPage.module.scss

```diff
@@ -10,11 +10,12 @@ main.wrapper {
 }
 
 section {
-  padding: 4rem 1rem 0 1rem !important;
+  padding: 4rem 0 0 !important;
 }
 
 .headerContainer {
   background-color: color.$light_gray_50;
+  padding: 0 1rem;
```

Comment:
Did a little bit of refactoring here since the inline padding was being applied to all the sections on the page.